simulation/vis_simulate.pro

```diff
@@ -51,7 +51,8 @@ FUNCTION vis_simulate,obs,status_str,psf,params,jones,skymodel,file_path_fhd=fil
       ENDELSE
 
     endfor
-    IF ~Keyword_Set(no_save) THEN save, file=init_beam_filepath, beam2_xx_image, beam2_yy_image, obs
+    test = strpos(init_beam_filepath, '_000_')
```

Comment:
I'm not sure what the purpose of this test is and if it's something generalised or specific to your testing.

Reply:
I think this was a temporary measure I added because I was running hundreds of simulations and they kept saving the same beam image (and I only wanted one copy). I'll just delete it.